prettier/src/index.ts

```diff
@@ -38,6 +38,8 @@ const tokenize = (input) => {
   const VARIABLE_REGEX = /({{.+?}})/gs;
   const HTML_TAG_WITH_HUBL_TAG_REGEX = /<[^>]*?(?={%|{{).*?>/gms;
   const STYLE_BLOCK_WITH_HUBL_REGEX = /<style.[^>]*?(?={%|{{).*?style>/gms;
+  const JSON_BLOCK =
+    /(?<={% widget_attribute.*is_json="?true"? %}|{% module_attribute.*is_json="?true"? %}).*?(?={%.*?end_module_attribute.*?%}|{%.*?end_widget_attribute.*?%})/gims;
```

Comment:
Quite a few patterns here for swapping hubl tokens for placeholders. I haven't seen any major performance issues with these yet but something to keep an eye out for.

Reply:
Questions for you! 
1. Is going through a list of regexs like this typically a performance concern? 
2. Is it something that _could_ be if the list of regex patterns grows too large? 
3. Are there other ways to do this that you're aware of that would make this more scalable from a performance perspective?